__tests__/app.test.js

```diff
@@ -101,8 +102,59 @@ describe("GET", () => {
         });
     });
 
-    // it("", () => {
-
-    // })
+    
   });
+
+  //task 5
+
+  describe("/api/articles", () => {
+ 
+    //200: responds with array of articles in descending order without body property
+
+    //error handling
+
+
+    //400: invalid sort by query
+
+    //
+
+
+    it("200: responds with array of article objects (correct amount) each with 8 properties", () => {
+      return request(app).get("/api/articles").expect(200).then((result) => {
+
+        expect(Object.keys(result.body.articles).length).toBe(13)
+        result.body.articles.forEach((article) => {
+          expect(Object.keys(article).length).toBe(7)
```

Comment:
We should use objectContaining to check every single property not just how many there are :)